### PR TITLE
support for case reference

### PR DESCRIPTION
**Pull request by jindrichskupa**
Hello Josef,
I added support for case references and legal form id (spisova znacka a umisteni + kod pravni formy). Please pull it into your upstream.

Thanks 

Jindra

### CODE REVIEW

File: lib/ruby-ares/parser.rb [hunk 1]
```diff
@@ -37,11 +44,23 @@ def self.get_subject(xml)
...
 
       # Create and return subject
-      return RubyARES::Subject.new(@ic, @dic, @name, @status, @addresses, @updated_at, @legal_form)
+      return RubyARES::Subject.new(@ic, @dic, @name, @status, @addresses, @updated_at, @legal_form, @legal_form_id, @case_reference)
     end
 
     protected
 
+      def self.find_case_reference(doc)
+        @case_reference = nil
```
Comment (strzibny) [inline lib/ruby-ares/parser.rb]: This is unnecessary as any new variable is `nil`.

File: lib/ruby-ares/parser.rb [hunk 2]
```diff
@@ -37,11 +44,23 @@ def self.get_subject(xml)
...
+        doc.find('D:SZ') do |node|
+          puts "jeeeem"
```
Comment (strzibny) [inline lib/ruby-ares/parser.rb]: This is most likely forgotten debug string ;).

**Comment by strzibny** [comment]
Thank you for the PR. This will need a CHANGELOG entry and perhaps an example in README would be nice too. I pointed out small issues in inline comments.

**Comment by jindrichskupa** [comment]
Ok, right, I send it without check after I have it done for my purpose.

**Comment by jindrichskupa** [comment]
I resolved merge conflict in README, added CHANGELOG record and fixup samo issues in code. I left up to you update of version number in gemspec .

**Comment by strzibny** [comment]
Thanks, merging.

**Pull request merged**